propagators/opentelemetry-propagator-grpc-census-binary/test/GrpcCensusPropagator.test.ts

```diff
@@ -272,3 +276,29 @@ function getUnsampledSpanEncodedArray() {
     0,
   ]);
 }
+
+function makeSetter(callback: any) {
+  const setter: TextMapSetter = {
+    set(carrier, key, value) {
+      callback(carrier, key, value);
+    },
+  };
+  return setter;
+}
+
+function makeGetter() {
+  const getter: TextMapGetter = {
+    get(carrier, key) {
+      if (carrier) {
+        return carrier.get(key);
+      }
+    },
+    keys(carrier) {
+      if (carrier == null) {
```

Comment:
Better to use `===` here in my opinion, so the code is explicit and to prevent confusion with coercion.

Reply:
This is exactly the same as `defaultTextMapGetter` the only difference was that I have to call "carrier.get" https://github.com/open-telemetry/opentelemetry-js/blob/master/packages/opentelemetry-api/src/context/propagation/TextMapPropagator.ts#L124

Reply:
@blumamir I agree that we should use strict checking by default however we didn't find a concensus on this, so for now we kinda allow both

Reply:
cool, it was just a suggestion. 
Using `==` works too :)